client/ayon_core/hosts/maya/plugins/publish/collect_render.py

```diff
@@ -290,8 +290,8 @@ def process(self, instance):
             "colorspaceView": colorspace_data["view"],
         }
 
-        rr_settings = context.data["project_settings"]["royalrender"]
-        if rr_settings["enabled"]:
+        rr_settings = context.data["project_settings"].get("royalrender", {})
```

Comment:
```suggestion
        rr_settings = context.data["project_settings"].get("royalrender")
```

Reply:
Maybe it would be better to use addons manager.
```
manager = context.data["ayonAddonsManager"]
if managet.get_enabled_addon("royalrender") is not None:
```

Reply:
ok, changed